Cabal-tests/tests/ParserTests/regressions/decreasing-indentation.check

```diff
@@ -0,0 +1,16 @@
+decreasing-indentation.cabal:40:3: Inconsistent indentation at lines 40, 41, 42, 43, 44, 50, 52, 57, 58, 70, 71
+No 'main-is' field found for executable warnings
+Deprecated extensions: 'PatternSignatures'. Instead of 'PatternSignatures' use 'ScopedTypeVariables'.
+'ghc-options: -O2' is rarely needed. Check that it is giving a real benefit and not just imposing longer compile times on your users.
+These packages miss upper bounds:
```

Comment:
I would suggest to fix those missing-upper-bounds warnings (actually: any warning that is not “decreasing indentation”). Makes the golden test clearer and more robust.

Reply:
The test file vanilla `RSA-1.0.0` file (with some extra additions at the end). I'd rather leave it as is, if this is not a hard requirement for PR acceptance.

Reply:
I agree that isolating the warning we test for is desirable. One reason is that if other warnings change in the future, we won’t have to touch this test result.

Why is it good (as seemingly implied) to have a vanilla cabal file of a random package in Cabal test suite? I don’t see any reason to strive for such files.

Reply:
Done